book/installation/docker.md

```diff
@@ -125,3 +125,13 @@ docker exec -it reth bash
 **If Reth is running with Docker Compose, replace `reth` with `reth-reth-1` in the above command**
 
 Refer to the [CLI docs](../cli/cli.md) to interact with Reth once inside the Reth container.
+
+## Run only Grafana in Docker
+
+This allows importing existing Grafana dashboards, without running reth in Docker.
+
+```bash
+docker compose -f etc/docker-compose.yml up -d --no-deps grafana
```

Comment:
for me to get this to work I had to comment out

https://github.com/paradigmxyz/reth/blob/cd0a2f34bcdd0e4ce86c915d3299848c3ba18ac7/etc/docker-compose.yml#L2

before I did that I got error
```
(root) Additional property name is not allowed
```

can we solve that in a nicer way than writing in the docs "comment out 'name'"? @shekhirin @rswanson

Reply:
hmm
```console
➜  reth (main) docker compose version
Docker Compose version v2.18.1

➜  reth (main) docker compose -f etc/docker-compose.yml up --no-deps -d grafana                                                                                                                                                                                                                                                                   ✭
[+] Building 0.0s (0/0)
[+] Running 1/0
 ✔ Container reth-grafana-1  Running
```

Reply:
I was using an outdated version of docker! thanks Alexey for catching that one too